osquery/CMakeLists.txt

```diff
@@ -144,6 +144,13 @@ TARGET_OSQUERY_LINK_WHOLE(run libosquery_additional)
 set_target_properties(run PROPERTIES COMPILE_FLAGS ${OS_COMPILE_FLAGS})
 install(TARGETS run DESTINATION bin)
 
+add_executable(osqueryctl main/ctl.cpp)
+TARGET_OSQUERY_LINK_WHOLE(osqueryctl libosquery_basic)
+TARGET_OSQUERY_LINK_WHOLE(osqueryctl libosquery_additional)
```

Comment:
You *could* nix this, it shouldn't need all the table implementations. But then it wouldn't have processes/users and you may want that.